conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -15,13 +15,37 @@
 from conda_store_server._internal import orm, schema
 from conda_store_server._internal.environment import filter_environments
 from conda_store_server._internal.server import dependencies
+from conda_store_server._internal.server.views.pagination import (
+    Cursor,
+    CursorPaginatedArgs,
+    Ordering,
+    OrderingMetadata,
+    paginate,
+)
 from conda_store_server.conda_store import CondaStore
 from conda_store_server.exception import CondaStoreError
 from conda_store_server.server import schema as auth_schema
 from conda_store_server.server.auth import Authentication
 from conda_store_server.server.schema import AuthenticationToken, Permissions
 
 
+def get_cursor(cursor: Optional[str] = None) -> Cursor:
+    return Cursor.load(cursor)
+
+
+def get_cursor_paginated_args(
```

Comment:
It looks like most of these `depends` type functions are in the [`conda_store_server/_internal/server/dependencies` ](https://github.com/conda-incubator/conda-store/blob/main/conda-store-server/conda_store_server/_internal/server/dependencies.py) module. I think it would be good to consolidate these into that module.

Reply:
Totally agree, thank you!